custom-xml-s2i/README.md

```diff
@@ -1,64 +1,72 @@
-# S2I for custom XML configuration
+# Loading Custom XML in Red Hat Data Grid for OpenShift
```

Comment:
Yeah, that's a fair point @wburns. This is specific to the RH DataGrid for OpenShift image so I think we can get away with it. Perhaps it would actually be better to host these demos in a different repo. In general I think the guidelines are to keep the Red Hat branding out of Infinispan docs. But worlds collide. For example I see lots of instances of Infinispan throughout the productized artifacts.

Reply:
These repo does not contain official docs. This org contains loosely coupled demos that exist out there, targeting community, product or both.

Reply:
Official product demos/quickstarts are in [jboss-developer/jboss-jdg-quickstarts](https://github.com/jboss-developer/jboss-jdg-quickstarts). Official community demos are really the [infinispan/infinispan-simple-tutorials](https://github.com/infinispan/infinispan-simple-tutorials).